### PR TITLE
Changed price of CAS flares from 10 to 5

### PR DESCRIPTION
**Pull request by Enukar**
## About The Pull Request

CAS flares is reduced from 10 to 5.
Deployable cameras changes from 3 for 6 points to 1 for 2 points
T-72 rpgs changes from 5 for 50 points to 1 for 10 points
Quickdeploy cades changes from 2 for 6 to 1 for 3
Fixed jetpack typo

## Why It's Good For The Game

Marines can get CAS flares for free by default, they should be cheaper even if nobody is gonna ever bother buying them anyway.
There is no reason to enforcer random quantitys of cades, rgs, cameras to be ordered, those should be ordered individually and up to the RO how many they want to order. 

## Changelog
:cl:
balance: CAS flares now cost 5 supply poinsts instead of 10.
balance: T-72 RPG can be ordered individually.
balance: Quickdeploy cades can be ordered individually.
balance: Deployable cameras can be ordered individually.
/:cl:

### CODE REVIEW

**Comment by Enukar** [comment]
cl
balance: CAS flares now cost 5 supply poinsts instead of 10.
/cl

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -948,9 +933,7 @@ CLOTHING
 
 /datum/supply_packs/clothing/jetpack
 	name = "Jetpack"
-	contains = list(
-		/obj/item/jetpack_marine,
-	)
+	contains = list(/obj/item/jetpack_marine,)
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:936]:
```suggestion
	contains = list(/obj/item/jetpack_marine)
```

File: code/modules/reqs/supplypacks.dm [hunk 2]
```diff
@@ -72,12 +72,8 @@ OPERATIONS
...
 /datum/supply_packs/operations/flares
-	name = "2 flare packs"
-	notes = "Contains 14 flares"
-	contains = list(
-		/obj/item/storage/box/m94,
-		/obj/item/storage/box/m94,
-	)
+	name = "flare packs"
+	contains = list(/obj/item/storage/box/m94)
 	cost = 2
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:78]:
```suggestion
	cost = 1
```

**Comment by Swagile** [comment]
should just remove CAS flares from supply packs entirely because Bravemole's PR made them virtually infinite

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**